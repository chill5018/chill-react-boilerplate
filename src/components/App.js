import React, { Component } from 'react'

class App extends Comment {
  render() {
    return (
      <div>
        <h1>Chill-start to React</h1>
      </div>
    )
  }
}

export default App