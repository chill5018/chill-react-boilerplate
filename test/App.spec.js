import App from '../src/components/App'
import React from 'react'
import { shallow } from 'enzyme'


describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<App/>)

    expect(wrapper.find('h1').text()).toBe('Chill-start to React')
    expect(wrapper).toMatchSnapshot()
  })
})
