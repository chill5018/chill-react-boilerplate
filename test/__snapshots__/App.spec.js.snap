// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Chill-start to React
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Chill-start to React",
      },
      "ref": null,
      "rendered": "Chill-start to React",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Chill-start to React
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Chill-start to React",
        },
        "ref": null,
        "rendered": "Chill-start to React",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
